pkgs/top-level/python-aliases.nix

```
@@ -72,21 +72,22 @@ mapAliases ({
   pytestquickcheck = pytest-quickcheck; # added 2021-07-20
   pytestrunner = pytest-runner; # added 2021-01-04
   python-lz4 = lz4; # added 2018-06-01
+  python_mimeparse = python-mimeparse; # added 2021-10-31
   python-subunit = subunit; # added 2021-09-10
   pytest_xdist = pytest-xdist; # added 2021-01-04
   python_simple_hipchat = python-simple-hipchat; # added 2021-07-21
   qasm2image = throw "qasm2image is no longer maintained (since November 2018), and is not compatible with the latest pythonPackages.qiskit versions."; # added 2020-12-09
   requests_toolbelt = requests-toolbelt; # added 2017-09-26
-  rotate-backups = throw "pythonPackages.rotate-backups was removed in favor of the top-level rotate-backups"; # added 2021-07-01
+  rotate-backups = throw "rotate-backups was removed in favor of the top-level rotate-backups"; # added 2021-07-01
   scikitlearn = scikit-learn; # added 2021-07-21
   selectors34 = throw "selectors34 has been removed: functionality provided by Python itself; archived by upstream."; # added 2021-06-10
   setuptools_scm = setuptools-scm; # added 2021-06-03
   smart_open = smart-open; # added 2021-03-14
   smmap2 = throw "smmap2 has been deprecated, use smmap instead."; # added 2020-03-14
   sphinxcontrib_plantuml = sphinxcontrib-plantuml; # added 2021-08-02
   sqlalchemy_migrate = sqlalchemy-migrate; # added 2021-10-28
-  topydo = throw "python3Packages.topydo was moved to topydo"; # added 2017-09-22
-  tvnamer = throw "python3Packages.tvnamer was moved to tvnamer"; # added 2021-07-05
+  topydo = throw "topydo was moved to topydo"; # added 2017-09-22
+  tvnamer = throw "tvnamer was moved to tvnamer"; # added 2021-07-05
```

Both comments don't make much sense from my point of view. Perhaps:

```suggestion
  topydo = throw "topydo was moved to pkgs.topydo"; # added 2017-09-22
  tvnamer = throw "tvnamer was moved to pkgs.tvnamer"; # added 2021-07-05
```

Or follow the comment for `rotate-backups`.